src/js/components/Select/doc.js

```diff
@@ -134,11 +134,9 @@ export const doc = Select => {
       children callback in order to render anything based on the current item.`,
     ).isRequired,
     open: PropTypes.bool.description(`Control the state of the component.`),
-    placeholder: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.node,
-      PropTypes.element,
-    ]).description('Placeholder text to use when no value is provided.'),
+    placeholder: PropTypes.node.description(
```

Comment:
I understand the clean up of `string`, but why did we decide to go with node rater than with element as the type?

Reply:
the node includes both the string and the element

Reply:
yea, but using simply `element` isn't enough?

Reply:
we totally can do element and string but just element will not cover string

Reply:
let's go with element & string rather than node, it will be more accurate.

Reply:
agree!